src/main/java/org/damap/base/rest/storage/InternalStorageTranslationDO.java

```diff
@@ -0,0 +1,27 @@
+package org.damap.base.rest.storage;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class InternalStorageTranslationDO {
+  private long id;
+  private long storageId;
+
+  @NotBlank(message = "language code cannot be blank")
+  @NotNull(message = "language code cannot be null") private String languageCode;
+
+  @NotBlank(message = "title cannot be blank")
+  @NotNull(message = "title cannot be null") private String title;
+
+  @NotBlank(message = "description cannot be blank")
```

Comment:
Can be empty

Reply:
I suppose you refer to the description, has been changed